src/Components/Publishing/Sections/ArtworkCaption.tsx

```diff
@@ -39,7 +43,7 @@ export class ArtworkCaption extends React.Component<ArtworkCaptionProps, null> {
     const { artwork: { artist, artists } } = this.props
 
     // Multiple artists
-    if (artists && artists.length > 0) {
```

Comment:
The `artists` array contains an `artist` even if there is just one.

Reply:
This conditional could be a const, something list `hasMultipleArtists` -- noticed you use it again on 207